test/parallel/feeLock.API.feePayment.parallel.test.ts

```diff
@@ -153,8 +153,10 @@ test("gasless- GIVEN a correct config for gasless swaps WHEN the user runs unloc
       await api.query.feeLock.accountFeeLockData(testUser1.keyRingPair.address)
     )
   );
-  const waitingBlock = accountFeeLockData.lastFeeLockBlock + periodLength;
-  await waitBlockNumber(waitingBlock, periodLength.toNumber() + 5);
+  const waitingBlock = await stringToBN(
```

Comment:
why await stringToBn?

Reply:
Because we need to add periodLength to understand which block we need to wait